apps/site/src/app/(home)/sections/MentorVolunteer/MentorVolunteer.tsx

```diff
@@ -0,0 +1,26 @@
+import Stand from "./Stand";
+import stand1 from "./stand-1.png";
+import stand2 from "./stand-2.png";
+
+const MentorVolunteer = () => {
+	return (
+		<div className="grid max-w-lg lg:max-w-none lg:grid-cols-[repeat(2,450px)] lg:items-stretch justify-center m-auto gap-x-8 gap-y-12 mt-24">
```

Comment:
Issue: The stands seem to be a little too close together compared to the design.